app/models/dance.rb

```diff
@@ -14,7 +14,9 @@ class Dance < ActiveRecord::Base
     elsif user.admin?
       all
     else
-      where('publish= true OR user_id= ?', user.id)
```

Comment:
SQLite doesn't have a boolean data type, so the hard-coded SQL doesn't parse.  For the SQLite adapter, Rails will map the values `true` and `false` to the strings `'t'` and `'f'` to accommodate.